packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -190,14 +197,18 @@ void WebView::ApplySettings(const flutter::EncodableMap& settings) {
           has_progress_tracking_ = std::get<bool>(value);
         }
       } else if (string_key == "debuggingEnabled") {
+        // Not supported on Tizen.
       } else if (string_key == "gestureNavigationEnabled") {
+        // Not implemented.
       } else if (string_key == "allowsInlineMediaPlayback") {
+        // Not applicable for Tizen (always allowed).
```

Comment:
This is true for the lightweight web engine. Is this also true for EWK?

Reply:
I think this feature only for iOS. and as far as I know, most weview implementations are allowed inline media playback by default. so I think you don't have to care about it.